src/components/SatelliteManifestPanel/SatelliteManifestPanel.tsx

```diff
@@ -68,6 +69,8 @@ const SatelliteManifestPanel: FunctionComponent<SatelliteManifestPanelProps> = (
   const [currentDetailUUID, setCurrentDetailUUID] = useState('');
   const [detailsDrawerIsExpanded, setDetailsDrawerIsExpanded] = useState(false);
   const [currentDetailRowIndex, setCurrentDetailRowIndex] = useState(null);
+  const [isDeleteConfirmationModalOpen, setIsDeleteConfirmationModalOpen] = useState(false);
+  const [currentDeletionUuid, setCurrentDeletionUuid] = useState('');
```

Comment:
We can go with caps.  I see it being recommended in a few style guides.  We always do it the way I have it in our ruby apps to prevent some things from looking weird, but it's okay.